docs/user-guide/binned-data/computation.ipynb

```diff
@@ -202,9 +213,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Arithmetic operations\n",
+    "## Arithmetic operations\n",
     "\n",
-    "Direct arithmetic operations with event data can be useful in some cases, but more commonly operations should act on event data as if it had been histogrammed are required.\n",
+    "Direct arithmetic operations with binned data can be useful in some cases, but more commonly operations should act on event data as if it had been histogrammed are required.\n",
```

Comment:
Sentence structure is off. Also uses binned data and event data in the same sentence. Unless there is some subtlety I am missing would be better to use a consistent descriptor for this kind of data within a sentence.

Reply:
Thanks for pointing this out, I think I am actually going to add a section, so no crystal ball is required for understanding what I had in mind when I wrote "direct arithmetic operations with event/binned data".